recipes/pyprojecttoolchain/conanfile.py

```diff
@@ -69,21 +69,54 @@ def context(self):
             "url": self._conanfile.url,
             "author": self._conanfile.author,
             "license": self._conanfile.license,
+            "description_file":  "README.md",
             "python_version": python_version
         }
 
 
+class ToolSipProjectPyQtBuilder(Block):
+    template = textwrap.dedent("""
+    {% if link_full_dll %}link-full-dll = true
+    {% endif %}py-pylib-dir = "{{ py_pylib_dir }}"
+    py-pylib-lib = "{{ py_pylib_lib }}"
+    {% if py_pylib_shlib is not none %}py-pylib-shlib = "{{ py_pylib_shlib }}"
+    {% endif %}qml-debug = {{ qml_debug | lower }}
```

Comment:
```suggestion
    {% if link_full_dll %}link-full-dll = true{% endif %}
    py-pylib-dir = "{{ py_pylib_dir }}"
    py-pylib-lib = "{{ py_pylib_lib }}"
    {% if py_pylib_shlib is not none %}py-pylib-shlib = "{{ py_pylib_shlib }}"{% endif %}
    qml-debug = {{ qml_debug | lower }}
```
Any reason why we don't do it like this? if, and end ifs on the same line for single line changes makes it more readable imo

Reply:
I'm fine with the suggested change if the generated toolchain still jas the correct format with respect to new lines

Reply:
Let's try this _after_ merge, so we know this wasn't the culprit when something goes wrong.